projects/1.20/assets/sculky-bits/sculkybits/lang/zh_cn.json

```
@@ -0,0 +1,49 @@
+{
+    "entity.sculkybits.sculk_saprophyte": "幽匿腐生者",
+    "item.sculkybits.sculk_saprophyte_spawn_egg": "幽匿腐生者生成蛋",
+    "entity.sculkybits.sculk_crawler": "幽匿爬行者",
+    "item.sculkybits.sculk_crawler_spawn_egg": "幽匿爬行者生成蛋",
+    "block.sculkybits.sculk_jaw": "幽匿颚",
+    "block.sculkybits.sculk_flesh": "幽匿肉",
+    "block.sculkybits.sculk_growth": "幽匿生长",
+    "block.sculkybits.sculk_cage": "幽匿牢笼",
+    "block.sculkybits.sculk_trap": "幽匿陷阱",
+    "block.sculkybits.sculk_lurker": "幽匿潜伏者",
+    "block.sculkybits.sculk_ribs": "幽匿肋骨",
+    "block.sculkybits.sculk_bones": "幽匿骨骼",
+    "block.sculkybits.sculk_stabber": "幽匿刺刀",
+    "block.sculkybits.sculk_absorber": "幽匿吸收者",
+    "block.sculkybits.sculk_lurer": "幽匿诱饵",
+    "block.sculkybits.sculk_feeler": "幽匿触感",
+    "block.sculkybits.sculk_nest": "幽匿巢穴",
+    "block.sculkybits.sculk_bone_block": "幽匿骨块",
+    "effect.sculkybits.sculk": "幽匿",
+    "item.minecraft.potion.effect.sculk_potion": "幽匿药水",
+    "item.minecraft.splash_potion.effect.sculk_potion": "幽匿溅射药水",
+    "item.minecraft.lingering_potion.effect.sculk_potion": "幽匿持续药水",
+    "item.minecraft.tipped_arrow.effect.sculk_potion": "幽匿箭矢",

```

```suggestion
    "item.minecraft.splash_potion.effect.sculk_potion": "喷溅型幽匿药水",
    "item.minecraft.lingering_potion.effect.sculk_potion": "滞留型幽匿药水",
    "item.minecraft.tipped_arrow.effect.sculk_potion": "幽匿之箭",
```
请注意参照原版的药水与药箭命名格式，全文通查
https://zh.minecraft.wiki/w/%E5%96%B7%E6%BA%85%E8%8D%AF%E6%B0%B4
https://zh.minecraft.wiki/w/%E6%BB%9E%E7%95%99%E8%8D%AF%E6%B0%B4
https://zh.minecraft.wiki/w/%E7%AE%AD#%E8%8D%AF%E7%AE%AD